alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultProviderDataAccessor.java

```diff
@@ -54,7 +53,8 @@
 
 @Component
 @Transactional
-public class DefaultProviderDataAccessor implements ProviderDataAccessor {
+public class DefaultProviderDataAccessor {
```

Comment:
Can this class be deleted since the BlackDuckProviderDataAccessor now exists.  And in the past DefaultProviderDataAccessor was the only ProviderDataAccessor implementation once we removed support for Polaris.

Reply:
We want to keep this class around along with the provider data tables. The changes I made to this class for now should render this class unused until we decide to delete it.

Reply:
I know, I was just pointing this out because Intellij can re-add those annotations in ~1 second.

Reply:
I do think we want to add `@Deprecated` here just to make sure people don't try to use this whether in tests or production.